dev-docs/source/ReleaseChecklist.rst

```diff
@@ -191,6 +191,7 @@ all of the Local Project Managers.
 1 week before Code Freeze
 #########################
 
+*  Check that all people with release roles are added to the *\#release-roles* Slack channel and others are removed.
```

Comment:
This is peak nitpick, but can we keep the paragraphs about the same length per line?
```suggestion
*  Check that all people with release roles are added to the *\#release-roles* 
   Slack channel and others are removed.
```

Reply:
Peak!

Reply:
I think my way formatted fine and wasn't beyond the line length according to pycharm! I've made the change anyway!

Reply:
Oh yeah, it didn't affect anything on the actual docs, just so it was the same as the surrounding code. That's why I almost felt bad mentioning it 😅